<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireWindow</class>
 <widget class="QMainWindow" name="WireWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>654</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image : url(:/pages.jpg)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="FromlineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid#F7D9C4;
border-radius: 7px;
}</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="TolineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid #F7D9C4;
border-radius: 7px;
}</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="AmountlineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid #F7D9C4;
border-radius: 7px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#F7D9C4;</string>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#F7D9C4;</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#F7D9C4;</string>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aldhabi</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OkButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
